multipass/reference/multipass-mount-command.md

```diff
@@ -6,7 +6,12 @@ Please don't copy it blindly, first update the URL passed to the form with the c
 
 > See also: [Mount](/t/28470), [How to share data with an instance](/t/27189), [`umount`](/t/27214), [ID mapping](/t/45986).
 
-The `multipass mount` command maps a local directory from the host to an instance, with the possibility to specify the [mount](/t/28470) type (classic or native) and define group or user [ID mappings](/t/45986).
+The `multipass mount` command maps a local directory from the host to an instance, with the possibility to specify the [mount](/t/28470) type (classic or native) and define group or user [ID mappings](/t/45986). Here's the syntax for mapping a local directory to your virtual machine using the classic mount:
```

Comment:
I like that you included a very specific example below! 

I'd add a like break to separate the example from the description:

```suggestion
The `multipass mount` command maps a local directory from the host to an instance, with the possibility to specify the [mount](/t/28470) type (classic or native) and define group or user [ID mappings](/t/45986). 

For example, here's the syntax for mapping a local directory to your virtual machine using the classic mount:
```

Reply:
That makes sense. Thanks @giuliazanchi